app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -96,6 +96,50 @@
                   :xs              => 'true',
                   :primary         => 'true'}
 
+%div{"ng-if" => (defined?(hostname_hide) ? false : true) && (defined?(fallback_hostnames) ? fallback_hostnames : false)}
+  .form-group{"ng-class" => "{'has-error': angularForm.#{prefix}_fallback_hostname1.$invalid}"}
+    %label.col-md-2.control-label{"for" => "#{prefix}_fallback_hostname1"}
+      = _('Fallback Hostname 1')
```

Comment:
Can this be `'Fallback AMQP Hostname 1'`?  (Same for `'Fallback Hostname 2'` below).

Reply:
Thanks @blomquisg. As discussed in gitter, the reason for `Fallback Hostname 1` was because this is already on the AMQP tab and the main input field is entitled `Hostname`.

The reason for adding the `hostname, ipv4 or ipv6` as a placeholder text is that with this long name, the field label is broken into two rows which looks ugly.